app/js/components/LabOrdersList.jsx

```diff
@@ -200,8 +201,8 @@ export class LabOrdersList extends PureComponent {
         <h1>
           <FormattedMessage
             id="app.labOrdersList.title"
-            defaultMessage="Welcome to LabOrdersList Page/Component"
-            description="Welcome header on LabOrdersList page" />
+            defaultMessage="Welcome to LabTestResult Page"
```

Comment:
This can just be "Lab Test Results", we don't need the "welcome" or "page", etc....